packager/hls/base/simple_hls_notifier.cc

```diff
@@ -41,6 +42,22 @@ bool IsCommonSystemId(const std::vector<uint8_t>& system_id) {
          std::equal(system_id.begin(), system_id.end(), media::kCommonSystemId);
 }
 
+std::string Base64EncodeUri(const std::string& prefix,
+                            const std::string& data) {
+    std::string data_base64;
+    base::Base64Encode(data, &data_base64);
```

Comment:
@rkuroiwa That is a good point. @Palethorn key_id can include arbitrary bytes, so it is possible that it may include illegal url chars after base64 conversion. (UUID is actually a form of hex representation of the data) Can you help us change the code here to use [Base64UrlEncode](https://cs.chromium.org/chromium/src/base/base64url.h?rcl=2491c29c85d9a50c2b6f51aece17343aa71061c2&l=30)? Thanks.